Kernel/FileSystem/Ext2FS/FileSystem.cpp

```diff
@@ -30,7 +31,23 @@ ErrorOr<void> Ext2FS::flush_super_block()
     MutexLocker locker(m_lock);
     VERIFY((sizeof(ext2_super_block) % logical_block_size()) == 0);
     auto super_block_buffer = UserOrKernelBuffer::for_kernel_buffer((u8*)&m_super_block);
-    return raw_write_blocks(2, (sizeof(ext2_super_block) / logical_block_size()), super_block_buffer);
+    auto const superblock_physical_block_count = (sizeof(ext2_super_block) / logical_block_size());
+
+    // First superblock is always at offset 1024 (physical block index 2).
```

Comment:
It's always at offset 1024, but isn't the block index dependent on `block_size()`?

Reply:
This is raw_write_blocks, so the block size is always 512 bytes.

Reply:
Neat!